public/js/clients/firefox.js

```diff
@@ -16,13 +16,26 @@ function getTabTarget(tab) {
   return TargetFactory.forRemoteTab(options);
 }
 
+function presentTabs(tabs) {
```

Comment:
`present` is a little ambiguous to me, it feels pretty close to `render` but this isn't actually presenting anything, it's just formatting. Not sure if there's a better word? If you want to keep it, that's fine!